[2023-08-15 20:27:57,409] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-02-06T00:00:00+00:00 [queued]>
[2023-08-15 20:27:57,415] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-02-06T00:00:00+00:00 [queued]>
[2023-08-15 20:27:57,415] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:27:57,415] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-15 20:27:57,415] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:27:57,439] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta_semana> on 2023-02-06 00:00:00+00:00
[2023-08-15 20:27:57,442] {standard_task_runner.py:52} INFO - Started process 51039 to run task
[2023-08-15 20:27:57,447] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta_semana', 'scheduled__2023-02-06T00:00:00+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/dados_climatics.py', '--cfg-path', '/tmp/tmp32z3ewb6', '--error-file', '/tmp/tmp41ymhvxl']
[2023-08-15 20:27:57,447] {standard_task_runner.py:80} INFO - Job 148: Subtask cria_pasta_semana
[2023-08-15 20:27:57,532] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-02-06T00:00:00+00:00 [running]> on host ubuntu.internal.cloudapp.net
[2023-08-15 20:27:57,599] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta_semana
AIRFLOW_CTX_EXECUTION_DATE=2023-02-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-06T00:00:00+00:00
[2023-08-15 20:27:57,600] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-15 20:27:57,601] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/guilherme/airflow/semana=2023-02/2023-02-13"']
[2023-08-15 20:27:57,608] {subprocess.py:85} INFO - Output:
[2023-08-15 20:27:57,610] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-15 20:27:57,646] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta_semana, execution_date=20230206T000000, start_date=20230815T202757, end_date=20230815T202757
[2023-08-15 20:27:57,699] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-15 20:27:57,716] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-15 20:39:44,490] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-02-06T00:00:00+00:00 [queued]>
[2023-08-15 20:39:44,496] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-02-06T00:00:00+00:00 [queued]>
[2023-08-15 20:39:44,496] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:39:44,496] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-15 20:39:44,496] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:39:44,519] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta_semana> on 2023-02-06 00:00:00+00:00
[2023-08-15 20:39:44,523] {standard_task_runner.py:52} INFO - Started process 56026 to run task
[2023-08-15 20:39:44,528] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta_semana', 'scheduled__2023-02-06T00:00:00+00:00', '--job-id', '249', '--raw', '--subdir', 'DAGS_FOLDER/dados_climatics.py', '--cfg-path', '/tmp/tmpb7v7u2tz', '--error-file', '/tmp/tmpfo25eiof']
[2023-08-15 20:39:44,529] {standard_task_runner.py:80} INFO - Job 249: Subtask cria_pasta_semana
[2023-08-15 20:39:44,586] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-02-06T00:00:00+00:00 [running]> on host ubuntu.internal.cloudapp.net
[2023-08-15 20:39:44,668] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta_semana
AIRFLOW_CTX_EXECUTION_DATE=2023-02-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-06T00:00:00+00:00
[2023-08-15 20:39:44,670] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-15 20:39:44,670] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/guilherme/airflow/dados/semana=2023-02/2023-02-13"']
[2023-08-15 20:39:44,677] {subprocess.py:85} INFO - Output:
[2023-08-15 20:39:44,680] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-15 20:39:44,714] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta_semana, execution_date=20230206T000000, start_date=20230815T203944, end_date=20230815T203944
[2023-08-15 20:39:44,781] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-15 20:39:44,799] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
